### PR TITLE
Adding Antrea fixes and 1.24 secret for serviceaccount

### PR DESCRIPTION
**Pull request by knabben**
### What this PR does / why we need it
A few fixes in the clusterclass package and Antrea installation script for Windows.

1. Antrea installation script does not work with `latest` forcing the current BOM version.
2. Adding a secret for the Kube-proxy ServiceAccount and being compatible with 1.24 release.
3. Disabling Windows automatic update service.

### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR
Created a 1.24 windows testbed locally with these changes.
```
successfully reconciled package: 'tk-antrea-v6-antrea' in namespace: 'tkg-system'
successfully reconciled package: 'tk-antrea-v6-vsphere-cpi' in namespace: 'tkg-system'
successfully reconciled package: 'tk-antrea-v6-vsphere-csi' in namespace: 'tkg-system'

Workload cluster 'tk-antrea-v6' created
```

### Release note

```release-note
Fixing Windows ClusterClass Antrea installation
```

### CODE REVIEW

File: packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml [hunk 1]
```diff
@@ -1502,6 +1502,12 @@ spec:
       - op: add
         path: /spec/template/spec/joinConfiguration/nodeRegistration/kubeletExtraArgs/register-with-taints
         value: os=windows:NoSchedule
+      - op: add
+        path: /spec/template/spec/files/-
+        value:
+          path: 'C:\k\prevent_windows_updates.ps1'
+          content: |
+            Set-Service -Name "wuauserv" -StartupType Disabled -Status Stopped
```
Comment (tenczar) [inline packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml:1510]: should this also be in `providers/infrastructure-vsphere/v1.4.1/cconly/base.yml`

File: packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml [hunk 2]
```diff
@@ -1502,6 +1502,12 @@ spec:
...
       - op: add
         path: /spec/template/spec/postKubeadmCommands/-
         value: powershell c:/k/prevent_windows_updates.ps1 -ExecutionPolicy Bypass
```
Comment (tenczar) [inline packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml:1513]: Is powershell ok with the forward slashes here? The file just added uses backslashes.

**Review by tenczar — CHANGES_REQUESTED** [review]
couple small comments.

File: packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml (hunk 2 above)
Comment (knabben) [reply packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml:1513]: yes, that's fine.

**Review by tenczar — APPROVED** [review]
LGTM

**Comment by jayunit100** [comment]
last bug for cluster class on windows, lets merge and iterate it forward !
/approve
/lgtm

**Review by jayunit100 — APPROVED** [review]

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: 0a307c9

Tests failed! Build no: 3310

**Pull request merged**